src/layout/strategy/boundingBox.js

```diff
@@ -13,7 +13,7 @@ export default function() {
             if (ty + d.height > bounds[1]) {
                 ty -= d.height;
             }
-            return {x: tx, y: ty};
+            return {height: d.height, width: d.width, x: tx, y: ty};
```

Comment:
This code is now in the following project: https://github.com/d3fc/d3fc-label-layout/blob/master/src/boundingBox.js

Would you mind applying this fix there?